app.go

```diff
@@ -224,3 +226,75 @@ func executeQuote(cmd command) bool {
 	// send the quote to the user
 	return true
 }
+
+func executeBuy(cmd command) bool {
+	//Gotta check users money and add a reserved portion
+	account := accountStore.GetAccount(cmd.UserID)
+
+	if account == nil {
+		log.Noticef("User %s does not have an account", account)
+	}
+
+	stockSymbol := cmd.Args[0]
```

Comment:
You're right. Depends entirely on how "non standard" we're going. Everything I've seen in the workload files has well-formed commands, errant spaces and `[###]` notwithstanding.